aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -440,7 +441,7 @@ module aptos_framework::voting {
     }
 
     #[test(aptos_framework = @aptos_framework, governance = @0x123)]
-    #[expected_failure(abort_code = 519)]
+    #[expected_failure(abort_code = 0x10002)]
```

Comment:
Just curious is there a better way to map these values like 0x10002 to the line of the code that fails now?

Reply:
Nothing changed here (yet). We do still have the same errmap functionality. What is better that one can now constructs a hex code naturally as `0xCCRRRR` (C for category, R for reason.)